tutorials/llm/llama-3/README.rst

```diff
@@ -1,63 +1,47 @@
 Llama 3 LoRA fine-tuning and deployment with NeMo Framework and NVIDIA NIM
```

Comment:
Llama 3 LoRA Fine-Tuning and Deployment with NeMo Framework and NVIDIA NIM

Reply:
Updated